### PR TITLE
Create release programatically and change docs

### PR DESCRIPTION
**Pull request by aCampello**
Closes https://github.com/wellcometrust/WellcomeML/issues/49

### CODE REVIEW

File: create_release.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+RELEASE_ID=$(curl -XGET --silent "https://api.github.com/repos/wellcometrust/WellcomeML/releases/tags/v$VERSION" | jq .id)
+
+cd dist/
+
+curl --request POST --silent --header "Authorization: token $GITHUB_TOKEN" -H "Content-Type: $(file -b --mime-type wellcomeml-$VERSION.tar.gz)" --data-binary @wellcomeml-$VERSION.tar.gz --url "https://uploads.github.com/repos/wellcometrust/WellcomeML/releases/$RELEASE_ID/assets?name=wellcomeml-$VERSION.tar.gz"
+curl --request POST --silent --header "Authorization: token $GITHUB_TOKEN" -H "Content-Type: $(file -b --mime-type wellcomeml-$VERSION.tar.gz)" --data-binary @wellcomeml-$VERSION.tar.gz --url "https://uploads.github.com/repos/wellcometrust/WellcomeML/releases/$RELEASE_ID/assets?name=wellcomeml-$VERSION-py3-none-any.whl"
+
+echo "Draft release created"
+echo "Please change the release description at https://github.com/wellcometrust/WellcomeML/releases/tag/v$VERSION"
```
Comment (nsorros) [inline create_release.sh:24]: Could we pick up the information from the changelog or commit message that updates the version? Or is it too much?

Comment (aCampello) [reply create_release.sh:24]: Maybe can adapt from this stack overflow

https://stackoverflow.com/questions/40450238/parse-a-changelog-and-extract-changes-for-a-version

Will look into it

File: Makefile [hunk 2]
```diff
@@ -37,6 +37,7 @@ dist:
 	$(VIRTUALENV)/bin/python3 setup.py sdist bdist_wheel
 	aws s3 sync dist/ s3://datalabs-packages/wellcomeml
 	aws s3 cp --recursive --acl public-read dist/ s3://datalabs-public/wellcomeml
+	chmod +x create_release.sh && ./create_release.sh
```
Comment (nsorros) [inline Makefile:40]: do you need to make it executable every time? does it work without it?

Comment (aCampello) [reply Makefile]: I removed this

**Comment by aCampello** [comment]
I was able to create a "danger" user input and removed chmod, however reading the release automatically from changelog proved to be a bit harder than I thought. Will look into it in the future, if I have time, but in the meantime we can go back and change the description MD ourselves.

**Pull request merged**